src/coprocessor/dag/expr/mod.rs

```diff
@@ -299,3 +347,8 @@ fn test_smoke() {
     let e: Result<Expression> = pb.try_into();
     let _ = e.unwrap();
 }
+
+#[cfg(test)]
+mod test {
+    pub use coprocessor::select::xeval::evaluator::test::{col_expr, datum_expr};
```

Comment:
Because after we rewrite all functions, I want to move those two functions to here. Now I list them here so that we can move them more easily in future.